### PR TITLE
[BULK UPDATE] DocuTune - Fix build validation issues: docs-link-absolute (part 1)

### PR DESCRIPTION
**Pull request by alexbuckgit**
**Global effort to fix build validation errors**

The Microsoft Skilling team is fixing build validation errors and specific brand inconsistencies in Microsoft technical documentation for the rest of FY23 Q1. This will help address various accessibility, security, and usability issues. This PR was generated using DocuTune. It includes only targeted fixes and minor formatting adjustments.

DocuTune PRs improve quality of technical content by finding and automatically correcting a broad set of issues only where the correction is suitable. Please review within five business days and merge or comment in the PR with any changes you'd like to see. Thank you!

CorrelationId: 0df9c2d6-63f8-4d1e-8b57-b33611a06514
InitiativeId: docutune-build-validation-issues-2022-05
PointOfContact: Alex Buck

### CODE REVIEW

**Comment by alexbuckgit** [comment]
#docutune-draft

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -89,7 +89,7 @@ If you're making substantial changes to an existing topic, adding or changing im
 
 5. If needed, make any necessary updates to your branch. Your pull request will be merged into the master branch after it has been reviewed and your changes are approved.
 
-On a certain cadence, we push all commits from the master branch into the live branch, after which you'll be able to see your contribution live at https://docs.microsoft.com/en-us/dynamics365/.
+On a certain cadence, we push all commits from the master branch into the live branch, after which you'll be able to see your contribution live at [Microsoft Dynamics 365 documentation](https://docs.microsoft.com/dynamics365)/.
```
Comment (annmariehitchcock) [inline CONTRIBUTING.md:92]: Parenthesis is out of place, should be (https://docs.microsoft.com/dynamics365/).

**Comment by annmariehitchcock** [comment]
#docutune-wait

**Comment by alexbuckgit** [comment]
Already handled via another repo.

#docutune-abandoned

**Pull request closed without merging**